optax/_src/alias_test.py

```diff
@@ -125,5 +125,24 @@ def test_explicit_dtype(self, dtype):
     self.assertEqual(expected_dtype, adam_state.mu.dtype)
 
 
+  def test_newton(self):
```

Comment:
Nit: I know most tests in this file don't do this, but in theory test names and/or docstrings should explain what the test checks. An example here could be `test_online_newton_step_decreases_loss(self)`, with a similar single-line docstring. This will be shown in the error message from the test when the tests fails and is therefore very helpful when debugging.